scalafix-core/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -1,7 +1,6 @@
 package scalafix.internal.patch
 
 import scala.annotation.tailrec
-import scala.collection.compat._
```

Comment:
This comes from the Scala3 run:
```
[warn] Failed to parse `-Wconf` configuration: origin=scala.collection.compat.*:s
```

Reply:
We will need to either revisit https://github.com/scalacenter/scalafix/pull/1797 or revert it

Reply:
I ended up patching `NoAutoTupling.scala` & `EscapeHatch.scala` as they didn't make a critical usage of scala.collection.compat.